app/templates/macros/toolkit_forms.html

```diff
@@ -17,7 +17,8 @@
     name=question_content.id,
     value=service_data[question_content.id],
     large=True,
-    max_length_in_words=question_content["maxLengthInWords"] or False
+    max_length_in_words=question_content["maxLengthInWords"] or False,
+    error=errors.get(question_content.id)['message'] or None
```

Comment:
Do we need the `or None` and `or False` here? Looking at the toolkit code (https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/forms/list-entry.html#L17) it just checks for `{% if error %}`, so there doesn't seem to be a need to replace any false value with `None`.